### PR TITLE
Fix: Fix all warnings in tests produced by Qiskit

### PR DESCRIPTION
**Pull request by SebastianMorawiec**
## Description

Include description of feature this PR introduces or a bug that it fixes. Include the following information:

Packages that depend on orquestra-qiskit are geting tons of warnings from qiskit. Fix those issues detected in this repo's tests

## Please verify that you have completed the following steps

- [x] I have self-reviewed my code.
- [ ] I have included test cases validating introduced feature/fix.
- [ ] I have updated documentation.

### CODE REVIEW

File: tests/orquestra/integrations/qiskit/conversions/pauli_conversions_test.py [hunk 1]
```diff
@@ -51,11 +49,9 @@ def test_paulisum_to_qiskitpauli():
 
     qiskit_op = qubitop_to_qiskitpauli(pauli_term)
 
-    ground_truth = (
-        PauliOp(Pauli("XZX"), 0.5) + PauliOp(Pauli("YZY"), 0.5)
-    ).to_pauli_op()
+    truth = SparsePauliOp.from_list([("XZX", 0.5), ("YZY", 0.5)])
```
Comment (AthenaCaesura) [inline tests/orquestra/integrations/qiskit/conversions/pauli_conversions_test.py:52]: Question: Why not use the standard constructor here. [link](https://docs.quantum.ibm.com/api/qiskit/qiskit.quantum_info.SparsePauliOp#:~:text=still%20ordered%20lexicographically.-,Example,Here%20is%20an%20example%20of%20how%20to%20use%20SparsePauliOp%20argsort.,-import%20numpy%20as)

File: tests/orquestra/integrations/qiskit/conversions/pauli_conversions_test.py [hunk 2]
```diff
@@ -51,11 +49,9 @@ def test_paulisum_to_qiskitpauli():
...
 
-    assert ground_truth == qiskit_op
```
Comment (AthenaCaesura) [inline tests/orquestra/integrations/qiskit/conversions/pauli_conversions_test.py:58]: Isn't ground truth the correct terminology here?

**Review by AthenaCaesura — APPROVED** [review]
looks good to me! Just some small questions. What happened to the first change that we made? I didn't see it in the review.

**Comment by SebastianMorawiec** [comment]
@AthenaCaesura is on holiday. @max-radin I have not merged this PR after review, as I did change makefile after initial review.

**Review by max-radin — APPROVED** [review]

**Pull request merged**